configure

```diff
@@ -552,9 +552,16 @@ eval VAL="\$${A}"
 echo " - ${A} = ${VAL}"
 done
 
-# This will not return correct values (tested):
+# These two lines only will not return correct values (tested),  glaring at at ACR..
 # echo " - ARCH = ${CPU}"
 # echo " - OSFAMILY = ${OS}"
 # Corrected (tested):
-echo " - ARCH = `uname -m|sed -e 's, ,,g'|cut -d - -f 1`"
-echo " - OSFAMILY = `uname -s|tr A-Z a-z`"
+# echo " - ARCH = `uname -m|sed -e 's, ,,g'|cut -d - -f 1`"
+# echo " - OSFAMILY = `uname -s|tr A-Z a-z`"
+
+# Beautified, refreshed shifted value for the need vars only:
+CPU="`uname -m|sed -e 's, ,,g'|cut -d - -f 1`"
```

Comment:
dont do this. change acr, or configure.acr if needed. i will evaluate the changes there